packages/contracts-periphery/test/contracts/L2/messaging/L2StandardERC721Factory.spec.ts

```diff
@@ -0,0 +1,115 @@
+/* External Imports */
+import { ethers } from 'hardhat'
+import { Signer, ContractFactory, Contract } from 'ethers'
+import {
+  smock,
+  MockContractFactory,
+  MockContract,
+} from '@defi-wonderland/smock'
+
+/* Internal Imports */
+import { expect } from '../../../setup'
+
+const L2_ERC721_BRIDGE_ADDRESS: string =
```

Comment:
What is this constant? Is it the contract address based on another file that deploys it?